flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-jdbc/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.flink</groupId>
+        <artifactId>flink-cdc-pipeline-connectors</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <properties>
+        <!-- JDBC connector for Flink 1.20 isn't released yet. -->
+        <flink.jdbc.connector.version>3.2.0-1.19</flink.jdbc.connector.version>
+    </properties>
+
+    <artifactId>flink-cdc-pipeline-connector-jdbc</artifactId>
```

Comment:
I think we should maintain the jdbc connector as `flink-connector-jdbc`. The jdbc pipeline connector here is not a connector as it does not contain a factory and we cannot find the factory by identifier `jdbc`.

Maybe we could rename this module as `flink-cdc-pipeline-connector-jdbc-parent`. And it should contain different modules as follows.
- flink-cdc-pipeline-connector-jdbc-parent
  - flink-cdc-pipeline-connector-jdbc-core
  - flink-cdc-pipeline-connector-jdbc-mysql